smoke/features/csi-driver-e2e.feature

```diff
@@ -0,0 +1,41 @@
+Feature: SharedSecrets and SharedConfigMap
+
+    As a user of openshift tech preview feature
+    I want to verify the functionality of shared resource of csi driver across the defined namespaces
+    To assure the stability in long run
+
+    Background:
+        Given We have an openshift techpreview cluster
+        And Shared resource csi driver is installed
+        Then create a project
+
+    @automated
+    Scenario: Configmaps are accessbile throughout the cluster : CSI-01-TC01
+        Given user has cluster scoped level permission to create CRD "sharedconfigmaps.sharedresource.openshift.io"
+        When user creates the configmap "share-config"   
+        And defines shared resource "my-shared-config" to share across all namespace
+        And creates another project to access configmap in different namespace
+        And RBAC for the service account to use the "configmap" in its pod
+        And creates a pod "my-csi-app-check"
+        And edits configMap "share-config" data
+        Then pod "my-csi-app-check" should reflect the changes
```

Comment:
so  you mention SharedSecrets up top, but there is no scenario that discussed creating a SharedSecret

Reply:
presumably that is coming "soon" @prietyc123 ?

Reply:
Yes @gabemontero  SharedSecret scenario will be added soon